test/services/cert/certs_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'katello_test_helper'
+require 'support/candlepin/owner_support'
+
+module Katello
+  class CertsTest < ActiveSupport::TestCase
+    def setup
+      VCR.insert_cassette('lib/tasks/verify_ueber_cert')
+      @org = get_organization
+      Resources::Candlepin::Owner.create(@org.label, @org.name)
+    end
+
+    def teardown
+      Resources::Candlepin::Owner.destroy(@org.label)
+      VCR.eject_cassette
+    end
+
+    def test_verify_ueber_cert
+      Setting.stubs(:[]).with(:ssl_ca_file).returns(File.join("#{Rails.root}", "/test/services/cert/helpers/ca.crt"))
```

Comment:
I'd also like to see a 'bad' case.  you might use the included redhat ca cert (ca/redhat-uep.pem) to test a bad caes